apps/site/test/site_web/controllers/stop_controller_test.exs

```diff
@@ -143,4 +144,58 @@ defmodule SiteWeb.StopControllerTest do
       end
     end
   end
+
+  describe "get_facilities/1" do
+    setup_with_mocks([
+      {SiteWeb.StopController, [:passthrough],
```

Comment:
**suggestion:** instead of mocking the function you're testing (which makes the unit test rather superficial), we should mock the underlying functions that the tested function depends on -- in this case, the `V3Api.Facilities.filter_by([{"stop", stop_id}])` call.